### ISSUE TITLE
'\n' breaks the output line

### ISSUE THREAD

**Issue by ghost** [dbcli/pgcli#446]
Hi! First of all thank you for awesome psql alternative)

I just noticed that '\n' symbol breaks the whole line

![pgcli_bad_line_breaks](https://cloud.githubusercontent.com/assets/1443953/12532321/a1bf64a2-c22a-11e5-83c1-453658efbd88.png)

but in psql it works fine

![psql_line_breaks_are_ok](https://cloud.githubusercontent.com/assets/1443953/12532394/7b1b35f4-c22c-11e5-807b-58717271bfc6.png)

I remember that somewhere there were an issue with similar problem, but don't remember exactly where, so I decided to create another one.
so what do you think about it?

**Comment by amjith** [comment]
Yes. This is a known issue. Until we replace the tabulate library with something better this will continue to be an issue. But I'll take this into consideration. Thanks!

**Comment by j-bennet** [comment]
Can't tell when this was fixed, but it works now as of `pgcli==2.1.0`:

```
postgres@/tmp:postgres> select E'abc\ndef', E'foo\nboo test'
+------------+------------+
| ?column?   | ?column?   |
|------------+------------|
| abc        | foo        |
| def        | boo test   |
+------------+------------+
SELECT 1
```
